src/cli/onefuzz/api.py

```diff
@@ -970,6 +971,33 @@ def list(
             results[container] = self.onefuzz.containers.files.list(container).files
         return results
 
+    def download(
+        self, job_id: UUID_EXPANSION, *, output: Optional[primitives.Directory] = None
+    ) -> None:
+        to_download = {}
+        tasks = self.onefuzz.tasks.list(job_id=job_id, state=None)
+        if not tasks:
+            raise Exception("no tasks with job_id:%s" % job_id)
+
+        for task in tasks:
+            for container in task.config.containers:
+                info = self.onefuzz.containers.get(container.name)
+                name = os.path.join(container.type.name, container.name)
+                to_download[name] = info.sas_url
+
+        if output is None:
```

Comment:
Is it correct that this PR just makes `output` default to the current directory?

If so, I suggest just editing the title or description to match the commit.

Reply:
no, this moves the implementation from `onefuzz debug job download_files` to jobs containers download.

In (probably only internally) discussions, we've stated the debug subset as not really stable.  This is an attempt to address one of the commands that is continuously useful.

Reply:
Ah! That's right.